Missionframework/modules/14_virtual/fnc/fn_virtual_preInit.sqf

```diff
@@ -49,7 +49,7 @@ if (isServer) then {
         params ["_object"];
         // Add built item to all curators
         {
-            _x addCuratorEditableObjects [[_object]];
+            _x addCuratorEditableObjects [[_object] + crew _object ];
```

Comment:
```suggestion
            _x addCuratorEditableObjects [[_object], true];
```
`curatorObj addCuratorEditableObjects [objects,addCrew]`
Ref: https://community.bistudio.com/wiki/addCuratorEditableObjects

Reply:
Forgot about that 😅